caampr-backend/src/main/java/com/jelistan/caampr/lambda/provider/GearListProvider.java

```diff
@@ -0,0 +1,34 @@
+package com.jelistan.caampr.lambda.provider;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBQueryExpression;
+import com.jelistan.caampr.lambda.model.Gear;
+
+import javax.inject.Inject;
+import java.util.List;
+
+public class GearListProvider implements GearProvider{
+
+    private final DynamoDBMapper mapper;
+
+    @Inject
+    public GearListProvider(DynamoDBMapper mapper){
+        this.mapper = mapper;
+    }
+
+    public List<Gear> getGearList(final String profileId){
```

Comment:
public methods should have valid javadocs

Reply:
added docs for methods